### ISSUE TITLE
[REQUEST] Report CPU frequency per core

### ISSUE THREAD

**Issue by frainz-de** [aristocratos/btop#190]
**Is your feature request related to a problem? Please describe.**
I was worried why my processor was at a low clock speed even though a process was using an entire core. Using `grep MHz /proc/cpuinfo` I noticed that the core in use was indeed at a much higher speed. It would be good to show this in btop.

**Describe the solution you'd like**
Instead of on top, the clock frequency is displayed next to the usage of each core.

**Describe alternatives you've considered**
Instead, we could write "avg" or something before the clock speed to make it more obvious that it is an average

**Comment by pc00per** [comment]
This is very much needed. CPU usage percentage is not same as per core frequency. This is needed to check whether frequencies are well within the range of idle frequency.

**Comment by fery25** [comment]
I'd really need it too.

**Comment by Mobmaker55** [comment]
This would be incredibly helpful!

**Comment by Faione** [comment]
This is very useful in heterogeneous systems, especially on ARM platforms. Is there any plan for it?

**Comment by djhunter67** [comment]
I agree, this feature would be greatly appreciated.  I came to the `issues` specifically to request this.

**Comment by smoluu** [comment]
Bump

**Comment by Shuddown** [comment]
I'd really like to see this as well! Would be useful for MacOS.

**Comment by kristof-dragon** [comment]
+1 for this feature. Would be usefule especially with the heterogeneous cores.

**Comment by AlekseyMelikov** [comment]
Necessary and useful feature

**Comment by FlavioMili** [comment]
Anyone ever started developing this? I could give it a try

**Comment by FlavioMili** [comment]
https://github.com/aristocratos/btop/pull/1593

if anyone wants to give a look and help me for other platforms :)
